test/spec/modules/eids_spec.js

```diff
@@ -304,15 +304,18 @@ describe('eids array generation for known sub-modules', function() {
   });
   it('originFloc', function() {
     const userId = {
-      flocId: {'id': 'Sample_Token'}
+      flocId: {id: 'Sample_Token', ver: 'chrome1.1'}
```

Comment:
is "sample_token" the origin trial token from the meta tag or the floc cohort id.  It should be the cohort id, and should probably have a name w/o token in it.